deployment/uberspace-ansible/files/hagrid.ini.j2

```diff
@@ -0,0 +1,18 @@
+[uwsgi]
+base = /home/{{ ansible_user }}/hagrid/hagrid
+chdir = /home/{{ ansible_user }}/hagrid
+
+http = :8000
+master = true
+wsgi-file = %(base)/wsgi.py
+touch-reload = %(wsgi-file)
+static-map = /static=%(base)/static
+
+app = wsgi
+
+#virtualenv = %(chdir)/venv
+
```

Comment:
It's quite important to inset the following line somewhere inside the
`[uwsgi]` section as processing media containing non-ASCII data may break the server. That actually happened  last Congress.
```suggestion
env = LANG=en_US.UTF-8
```

Reply:
(Media doesn't strictly mean images)

Reply:
isn't that set in the `settings.py`?

Reply:
You need both. In uwsgi for the input passing and in settings.py for python internal processing.